Resources/Locale/en-US/administration/ui/manage-solutions/manage-solutions.ftl

```diff
@@ -1,4 +1,9 @@
 admin-solutions-window-title = Solution Editor - {$targetName}
 admin-solutions-window-solution-label = Target solution:
 admin-solutions-window-add-new-button = Add new reagent
-admin-solutions-window-capacity-label = Capacity {$currentVolume}/{$maxVolume}u
+admin-solutions-window-volume-label = Volume {$currentVolume}/{$maxVolume}u
+admin-solutions-window-capacity-label = Capacity (u):
+admin-solutions-window-specific-heat-label = Specific Heat: {$specificHeat} J/(K*u)
+admin-solutions-window-heat-capacity-label = Heat Capacity: {$heatCapacity} J/K
+admin-solutions-window-temperature-label = Temperature (K):
+admin-solutions-window-thermal-energy-label = Thermal Energy (J):
```

Comment:
might want to look into the generic unit formatting in localization for some of the SI units here, but not strictly necessary